src/Model/CompositeTypeGo.cs

```diff
@@ -250,7 +250,10 @@ public string Fields()
                 {
                     // embed as an anonymous struct.  note that the ordering of this clause is
                     // important, i.e. we don't want to flatten primary types like dictionaries.
-                    indented.AppendFormat("*{0} {1}\n", property.ModelType.Name, property.JsonTag());
+                    indented.AppendFormat((property.ModelType as CompositeTypeGo).IsPolymorphic ?

```

Comment:
Can you please update the comment explaining why we might need to emit as a pointer.

Reply:
Fixed